### ISSUE TITLE
PiHole crashes the 8 frames LED you have to reboot after enabling PiHole

### ISSUE THREAD

**Issue by dansche** [Qrome/marquee-scroller#167]
Don't know why, but PiHole let crash the 8 segment scrolling and you have to reboot.

Any idea?

**Comment by Qrome** [comment]
Yeah, I think it is limited to the double-wide. I have not tested it on anything larger than a double-wide. There might be a limitation.
